ext/tiny_tds/extconf.rb

```diff
@@ -157,7 +157,7 @@ def configure
         end
         args = [ "CFLAGS=-DDSO_WIN32",
             "./Configure",
-            "no-shared",
+            "shared",
```

Comment:
I'm really not sure about the implications of this change. The resulting binaries have different exports, different sizes, and different symbols, and use a different build process than the old `dllwrap`. Given the comment below I'm not sure how safe this is. Any ideas?